src/main/client/batch_operate.c

```diff
@@ -108,11 +108,13 @@ static bool batch_operate_cb(const as_batch_result *results, uint32_t n,
  * @param py_ops                    The list containing op dictionaries.
  * @param py_policy_batch      		Python dict used to populate policy_batch.
  * @param py_policy_batch_write     Python dict used to populate policy_batch_write.
+ * @param py_ttl                    TTL value to set for each record.
  *******************************************************************************************************
  */
 static PyObject *AerospikeClient_Batch_Operate_Invoke(
     AerospikeClient *self, as_error *err, PyObject *py_keys, PyObject *py_ops,
-    PyObject *py_policy_batch, PyObject *py_policy_batch_write)
+    PyObject *py_policy_batch, PyObject *py_policy_batch_write,
+    PyObject *py_ttl)
```

Comment:
Why is this ttl needed if `py_policy_batch_write` already has a ttl field?

Reply:
The batch write policy's ttl is only supposed to be set in a batch write policy in the client's config. See my comment above: https://github.com/aerospike/aerospike-client-python/pull/536#discussion_r1387001831